jans-cedarling/cedarling_pg/tests/pg_regress/sql/setup.sql

```
@@ -0,0 +1,15 @@
+-- Install / drop / reinstall round-trip. Verifies catalog creation and clean teardown.
+CREATE EXTENSION cedarling_pg;
+
+-- Catalog schema should exist after CREATE EXTENSION.
+SELECT nspname FROM pg_namespace WHERE nspname = 'cedarling';
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)